app/views/jobseekers/accounts/show.html.haml

```diff
@@ -23,7 +23,7 @@
             = t(".summary_list.change")
             %span.govuk-visually-hidden
               = t(".summary_list.password")
-    = button_to "Temporary log out button until we create a proper logout link in the header", destroy_jobseeker_session_path, method: :delete
+    = button_to "Temp log out button pending logout link in the header", destroy_jobseeker_session_path, method: :delete
```

Comment:
just made text bit shorter so it doesnt throw mobile view alignment out

Reply:
I think you can just delete that line now the nav has been updated for jobseekers

Reply:
doh, of course i can, ive been signing out in new nav all day!